service/src/main/java/uk/gov/hmcts/reform/fpl/service/validators/CaseSubmissionChecker.java

```diff
@@ -59,7 +59,7 @@ private List<Event> getRequiredEvents(CaseData caseData) {
 
         events.add(CASE_NAME);
         events.add(ORDERS_SOUGHT);
-        if (!caseData.isC1Application() 
+        if (!caseData.isC1Application()
```

Comment:
revert this change? Seems no difference.

Reply:
done